src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import * as S from "./styles";
+
+const profileContent: React.FC = () => {
+    const [name, setName] = useState<string | null>(null);
```

Comment:
What happens if instead of <string | null >, I use <string> instead only (btw you should still use string | null to explicitly state this value can be null)

Reply:
I assume its <string> only, in which case it would still be fine maybe?

Reply:
i think my bad harusny I use string only, but iya its fine since null subset ny string